tiledb/api/c_api/buffer/test/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# tiledb/api/c_api/buffer/test/CMakeLists.txt
+#
+# The MIT License
+#
+# Copyright (c) 2022 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+find_package(Catch_EP REQUIRED)
+
+add_executable(unit_capi_buffer EXCLUDE_FROM_ALL)
+target_sources(unit_capi_buffer PUBLIC unit_capi_buffer.cc)
+target_include_directories(unit_capi_buffer PRIVATE ${TILEDB_EXPORT_HEADER_DIR})
```

Comment:
The export header should be declared on `unit_capi_buffer` so that anything that requires it has the right include path set automatically.

Reply:
Removed and unneeded if I properly depend on the `capi_buffer_stub` object library. Fixed now.